tests/containers/netavark_integration.pm

```diff
@@ -0,0 +1,81 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: netavark aardvark-dns
+# Summary: Upstream netavark integration tests
+# Maintainer: QE-C team <qa-c@suse.de>
+
+use Mojo::Base 'containers::basetest';
+use testapi;
+use serial_terminal qw(select_serial_terminal);
+use utils qw(script_retry);
+use containers::common;
+use containers::bats qw(install_bats enable_modules);
+use version_utils qw(is_sle);
+
+my $test_dir = "/var/tmp";
+my $netavark_version = "";
+
+sub run_tests {
+    my %params = @_;
+    my ($skip_tests) = ($params{skip_tests});
+
+    return if ($skip_tests eq "all");
+
+    my $log_file = "netavark.tap";
```

Comment:
```suggestion
    my $log_file = "netavark_tap.log";
```
The `tap` file extension might be misleading. I would rename it.

Reply:
> The `tap` file extension might be misleading. I would rename it.

It needs to be `.tap` for the TAP processor.

Reply:
ah ok